### PR TITLE
.net core 3.0 in maintenance mode and end of support will be in march 2020

### PR DESCRIPTION
**Pull request by SychevIgor**
https://dotnet.microsoft.com/platform/support/policy/dotnet-core

### CODE REVIEW

File: test/TestApp.AspNetCore.3.1/TestApp.AspNetCore.3.1.csproj [hunk 1]
```diff
@@ -11,16 +11,22 @@
...
     <ProjectReference Include="..\..\src\OpenTelemetry.Api\OpenTelemetry.Api.csproj" />
     <ProjectReference Include="..\..\src\OpenTelemetry.Collector.AspNetCore\OpenTelemetry.Collector.AspNetCore.csproj" />
     <ProjectReference Include="..\..\src\OpenTelemetry.Collector.Dependencies\OpenTelemetry.Collector.Dependencies.csproj" />
-    <ProjectReference Include="..\..\src\OpenTelemetry.Exporter.Zipkin\OpenTelemetry.Exporter.Zipkin.csproj" />
     <ProjectReference Include="..\..\src\OpenTelemetry.Hosting\OpenTelemetry.Hosting.csproj" />
     <ProjectReference Include="..\..\src\OpenTelemetry\OpenTelemetry.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="coverlet.msbuild" Version="2.8.0">
```
Comment (bruno-garcia) [inline test/TestApp.AspNetCore.3.1/TestApp.AspNetCore.3.1.csproj]: nit: any reason to change this? It's a test project so what it exposes from the dependency isnt' really relevant here.

Comment (SychevIgor) [reply test/TestApp.AspNetCore.3.1/TestApp.AspNetCore.3.1.csproj]: you are totally right. it's not .net core 3.1 specific thing. Missed it when review my changes. 
Should I rollback it or we can skip it for now?

Comment (SergeyKanzhelev) [reply test/TestApp.AspNetCore.3.1/TestApp.AspNetCore.3.1.csproj]: please rollback

Comment (SychevIgor) [reply test/TestApp.AspNetCore.3.1/TestApp.AspNetCore.3.1.csproj]: done

**Pull request merged**